src/student_player/PentagoBitBoard.java

```diff
@@ -345,7 +345,7 @@ long getRandomMove() {
 					byte[][] swapsQ0 = new byte[4][2];
 					int initialCapacityQ0 = (QUAD_SIZE * QUAD_SIZE) * swaps.length;
 
-					Q0 = equalQuadrants.get(0).get(0);
+					Q0 = equalQuadrants.get(0);
```

Comment:
Shouldn't this be `Q0 = uniqueQuadrant.get(0)` ?

Reply:
You're very correct, I managed to mess up the on line fix haha